index.html

```diff
@@ -0,0 +1,231 @@
+        skills.
+      </p>
+      <div class="main-time">
+        <h3 class="main-sub-heading">Registration starts 2022.10.15(MON)</h3>
+        <a class="zoom" href="#">Register now for Zoom link</a>
+      </div>
+    </section>
+
+    <section class="main-section2">
+      <img src="chef.jpg" alt="chef" class="chef-img" />
+      <h2 class="headline">Welcome to the Institute of</h2>
+      <h3 class="hl2">Culinary Education</h3>
+      <h4 class="hl3">Ranked No.1 Culinary School in South America*</h4>
+
+      <p class="para2">
+        The Institute of Culinary Education is a world-class culinary school
+        located in the heart of Santiago, Chile. We offer a variety of programs
+        to help you achieve your culinary goals, whether you’re looking to
+        become a professional chef, start a new career, or just expand your
+        cooking skills.
+      </p>
+      <div class="main-time">
+        <h3 class="main-sub-heading">Registration starts 2022.10.15(MON)</h3>
+        <a class="zoom" href="#">Register now for Zoom link</a>
+      </div>
+    </section>
+
+    <section id="program" class="middle-section">
+      <h2 class="program">The Program</h2>
+
+      <div class="middle-section-container">
+        <div class="middle-section-container2-1">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">Professional Chef Program</h3>
+          <p class="middle-section-para">
+            The Professional Chef Program is a 12-month, full-time program that
+            will teach you the culinary skills and business knowledge you need
+            to succeed in the culinary industry.
+          </p>
+        </div>
+        <div class="middle-section-container2-2">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">
+            Professional Pastry Program
+          </h3>
+          <p class="middle-section-para">
+            The Professional Pastry Program is a 12-month, full-time program
+            that will teach you the culinary skills and business knowledge you
+            need to succeed in the culinary industry.
+          </p>
+        </div>
+        <div class="middle-section-container2-3">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">
+            Professional Culinary Program
+          </h3>
+          <p class="middle-section-para">
+            The Professional Culinary Program is a 12-month, full-time program
+            that will teach you the culinary skills and business knowledge you
+            need to succeed in the culinary industry.
+          </p>
+        </div>
+      </div>
+
+      <button type="button" class="btn">See the full program</button>
+    </section>
+    <div class="teacher-topic">
+      <h2 class="teacher">Featured Teachers</h2>
+    </div>
```

Comment:
- [ ] [OPTIONAL] You have addressed the layout issue mentioned by previous reviewers, but your `Teacher boxes` have a lot of space on the `right` and `left` sides, because of this, you have made the elements in the `Teachers` section smaller. Please correct your mistake by looking at the picture below!

![image](https://user-images.githubusercontent.com/63915024/191833903-58cd322b-097c-4da5-b3fe-d80b2a204ea5.png)